src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/resource/model/StepRequestBuilder.kt

```diff
@@ -18,8 +19,21 @@ fun anotherValidCreateStepRequest(
   title: String = "Complete communication skills course",
   targetDateRange: TargetDateRange = THREE_TO_SIX_MONTHS,
   sequenceNumber: Int = 2,
-): CreateStepRequest =
-  CreateStepRequest(
+): StepRequest =
+  StepRequest(
+    title = title,
+    targetDateRange = targetDateRange,
+    sequenceNumber = sequenceNumber,
+  )
+
+fun aValidUpdateStepRequest(
```

Comment:
I've added this to show how we could use this builder for creating and updating a Step.